app/models/item.rb

```diff
@@ -0,0 +1,6 @@
+class Item < ApplicationRecord
+  validates :title, presence: true
+  validates :description, presence: true
+
+  # belongs_to :user
```

Comment:
Generally best practice is not to commit code comments like these. However, hopefully you'll be adding a `User` model soon, so we can let this one slide.